server/app/schema.py

```diff
@@ -55,11 +57,20 @@ class Meta:
         interfaces = (GlobalNode,)
 
 
+class ItemTranslation(SQLAlchemyObjectType):
+    class Meta:
+        model = ModelItemTranslation
+        interfaces = (GlobalNode,)
+
+
 class Item(SQLAlchemyObjectType):
     stats = graphene.NonNull(
         graphene.List(graphene.NonNull(ItemStats))  # Use list instead of connection
     )
     conditions = graphene.NonNull(graphene.List(graphene.NonNull(ItemCondtions)))
+    item_translations = graphene.NonNull(

```

Comment:
I'm not sure how you plan to use this, but we should be able to guess the user's language using Accept-Language in `info.context.headers`

Reply:
So you want to query for the specific translation and only return that?

Reply:
Yeah, I think so, have you found use cases for querying for translations?